jdbc-sink/src/main/java/org/springframework/cloud/stream/module/jdbc/DefaultInitializationScriptResource.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.jdbc;
+
+import java.nio.charset.Charset;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.core.io.ByteArrayResource;
+
+/**
+ * An in-memory script crafted for dropping-creating the table we're working with.
+ * All columns are created as VARCHAR(2000).
+ *
+ * @author Eric Bottard
+ */
+public class DefaultInitializationScriptResource extends ByteArrayResource {
```

Comment:
Any arguments that this class should be `public`, but not an inner one in the `JdbcSinkConfiguration` ?